whitepaper/use-case-sequuence-diagram-blockchain-migration.puml

```diff
@@ -0,0 +1,78 @@
+@startuml Sequence Diagram - End User Wallet Authentication Authorization
```

Comment:
@RafaelAPB is that the correct title? ;-)

Reply:
That was not the correct file. Fixed!